DANSEWELL - Reno

```diff
@@ -0,0 +1,11 @@
+Reno NV
+
+Resturaunts  (Spell-Check is not automatic)
```

Comment:
@trjnsteve : they are OK.  better for shows if something good comes into town.  I don't go into Casinos very often.